editor/code_editor.cpp

```diff
@@ -836,6 +870,13 @@ void CodeTextEditor::_line_col_changed() {
 	sb.append(itos(positional_column + 1).lpad(3));
 
 	line_and_col_txt->set_text(sb.as_string());
+
+	if (find_replace_bar) {
+		if (!find_replace_bar->line_col_changed_for_result)
+			find_replace_bar->needs_to_count_results = true;
+
+		find_replace_bar->line_col_changed_for_result = false;
+	}
```

Comment:
Probably need this under `_text_changed` as well, as changes via code will not always move the caret.

Reply:
Oh... right.. I'm gonna do it for text changed too.